CMakeLists.txt

```diff
@@ -391,6 +391,25 @@ endif()
 #)
 
 
+# CI Helper ###################################################################
+#
+
+# calling `python -m pip wheel .` and install re-using the build directory
+# note: this is mainly for our CI Python regression scrips, users and package
+#       managers should just use `python -m pip wheel .` and `... install *whl`
+#       directly
+set(PYINSTALLOPTIONS "" CACHE STRING "Additional parameters to pass to `pip install`")
+add_custom_target(install_pip
+    ${CMAKE_COMMAND} -E env PYWARPX_LIB_DIR=${CMAKE_LIBRARY_OUTPUT_DIRECTORY} python3 -m pip wheel -v --use-feature=in-tree-build ${WarpX_SOURCE_DIR}
+    COMMAND
+        python3 -m pip install --force-reinstall -v ${PYINSTALLOPTIONS} ${CMAKE_BINARY_DIR}/*whl
```

Comment:
Just to confirm: so we don't need a `--user` here, correct?

Reply:
That's correct. Since #2653 we run out tests (incl. GNUmake before the current PR) in a virtual environment.

`--user` was only needed when modifying the user-local Python environment. That was a bit unexpected/had side-effects and is solved now.